tao_compiler/mlir/disc/transforms/codegen_utils.cc

```diff
@@ -30,11 +30,13 @@ namespace disc_ral {
 int getRowReductionScheduleHint(Operation* op) {
   assert(op);
   lmhlo::FusionOp fusion = op->getParentOfType<lmhlo::FusionOp>();
-  // Use schedule 2 by default.
-  if (!fusion) return DISC_WARP_WISE_ROW_REDUCE;
+  // Use schedule 1 by default.
+  // Schedule 1 has a better performance in a wider range of shapes than
+  // schedule 2.
```

Comment:
Just curiouse about the insight behind this change...

Reply:
After a wide range of experiments, I find one-block-one-row usually has a better performance than one-warp-one-row schedule for most cases.